packages/examples/nodejs/.yarnrc.yml

```diff
@@ -0,0 +1 @@
+yarnPath: .yarn/releases/yarn-1.22.19.cjs
```

Comment:
what if the yarn version installed is different?

Reply:
This automatically switch to Yarn1 if you have yarn >1 and is ignored if you use yarn1.